aiida/repository/control.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Module for overall repository control commands."""
```

Comment:
Ok, I can move it there. However now I am not sure what is the hierarchy of concepts here, and (combined with the other comment on the `repository_backend` jargon) what exactly is what we call the backend. The way I understood it before, I thought the idea was that it would be something like this:

 - Storage Backend
   - Database Backend
   - Repository Backend

But now from these comments I understand it is more like:

 - Storage
   - Database Backend ("**the** backend")
     - Repository

Is this so? Is there any reason why the database and repository can't be "independent" from one another?